pkg/daemon/update.go

```diff
@@ -464,6 +486,31 @@ func getFileOwnership(file ignv2_2types.File) (int, int, error) {
 	return uid, gid, nil
 }
 
+// Update a given PasswdUser's SSHKey
+func (dn *Daemon) writeSSHKeys(newUsers []ignv2_2types.PasswdUser) error {
+	sshDirPath := "/home/core/.ssh"
```

Comment:
`core` here should come from the username in the spec, right?

Reply:
Yep, I was going to move some of my logic from my older PR over https://github.com/openshift/machine-config-operator/blob/3eff300a83970775dcc2ede2b17d17752bb4c535/pkg/daemon/update.go#L348

Reply:
In the future it would come from HomeDir but we don't support Users at this time.

So I will use the name for now and make a note.